lxd/instance.go

```diff
@@ -148,9 +148,16 @@ func instanceCreateFromImage(s *state.State, r *http.Request, img *api.Image, ar
 	revert.Add(cleanup)
 	defer instOp.Done(nil)
 
-	err = s.DB.Cluster.UpdateImageLastUseDate(args.Project, img.Fingerprint, time.Now().UTC())
+	err = s.DB.Cluster.Transaction(context.TODO(), func(ctx context.Context, tx *db.ClusterTx) error {
```

Comment:
I'm working in the same area of the code base now, so please can you put these changes up in a separate PR so we can merge sooner rather than later and avoid a conflict later.